plugin/lsc.vim

```diff
@@ -44,6 +44,8 @@ command! LSClientDisable call lsc#server#disable()
 command! LSClientEnable call lsc#server#enable()
 command! LSClientDisableDiagnosticHighlights call <SID>DisableHighlights()
 command! LSClientEnableDiagnosticHighlights call <SID>EnableHighlights()
+command! LSClientFormat call lsc#format#formatFile()
+command! -range LSClientFormat '<,'> call lsc#format#formatRange()
```

Comment:
This overrides the `:LSClientFormat` that calls `formatFile()`; running it doesn't work:

```
:100verbose LSClientFormat
Executing: '<,'> call lsc#format#formatRange()
E20: Mark not set
```

If I do use visual mode to set `<'` and `'>` I get:

    [lsc:Error] JSON RPC method not found: "RangeFormatting" not yet implemented

I guess my LSP server doesn't implement that.

Anyway, if I comment out this line then it works. Probably need to be a bit smarter about the range being passed and what mode it's in.